### PR TITLE
[DOCS-409] Dedicated Page for Beta Integrations

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
### Added

| Page | Deploy Preview | Comment |
| ----- | ----- | ----- |
| Request a Beta Integration | https://deploy-preview-316--cobalt-docs.netlify.app/integrations/beta/ | Created a dedicated page for beta integrations |

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Integrations/Beta/_index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+1. Navigate to the **Integrations** page, and select the desired app with the {{< beta-label >}} label.
+1. Complete the sign-up form.
+
+By submitting a request, you're joining a waiting list to have the integration enabled for your next pentest with Cobalt.
+
+![Request a beta integrations from the Integrations page](/integrations/BetaIntegrationsUI.png "Request a beta integrations from the Integrations page")
+
+## Beta Program Stages
+
+The beta program consists of the following stages:
```
Comment (eugenerevzin) [inline content/en/Integrations/Beta/_index.md:31]: > Once you've submitted the request for a beta integration, the process will consist of the following stages:

wdyt?

**Review by eugenerevzin — APPROVED** [review]
@eugenerevzin Thanks for your feedback!

How about this wording?

Once you've submitted a request, you go through the following stages of the beta program:

**Comment by ana-dashuk-cobalt** [comment]
@eugenerevzin Thanks for your feedback!

How about this wording?

Once you've submitted a request, you go through the following stages of the beta program:

**Comment by ana-dashuk-cobalt** [comment]
@eugenerevzin We can also add a call-to-action inviting users to get in touch. Wdyt? The button link will be set to `href="mailto:integrations@cobalt.io"`. Wdyt?


Screenshot 2023-03-14 at 11 22 30

**Comment by eugenerevzin** [comment]
> @eugenerevzin We can also add a call-to-action inviting users to get in touch. Wdyt? The button link will be set to `href="mailto:integrations@cobalt.io"`. Wdyt?
> 
> Screenshot 2023-03-14 at 11 22 30

@ana-dashuk-cobalt - Love the idea of the CTA.  Should we maybe add a line above the button, in bold, with something to the effect of:  "Want to try a beta integration? Navigate to the Integrations tab in the Cobalt platform, and select an app with the Beta label." 

Then below, have the button say "Want to learn more? Get in touch".  

Thinking it might be more actionable to direct them to go to the app up front, but also provide the email inquiry option. wdyt?

**Comment by ana-dashuk-cobalt** [comment]
@eugenerevzin Let's try the following:

- I've added a link to the Submit a Request section to the note at the top of the page.
- I've highlighted this sentence using a different color: `Navigate to the Integrations page in the Cobalt app to get started.`
- Let's use a different label for the button. `Learn more` would work for an in-app modal, but we're already on the documentation page with extensive information. So I suggest `Have Questions? Contact Us »`

Screenshot 2023-03-14 at 14 49 46

**Comment by eugenerevzin** [comment]
@ana-dashuk-cobalt - looks great!
